### ISSUE TITLE
 fviz_dend(): `guides(<scale> = FALSE)` is deprecated

### ISSUE THREAD

**Issue by facorread** [kassambara/factoextra#141]
`guides(<scale> = FALSE)` is deprecated. Please use `guides(<scale> = "none")` instead. 

You might want to review the following lines:
[fviz_mclust.R:46](https://github.com/kassambara/factoextra/blob/1689fc74b938510b773e4a595999e82bdf37104e/R/fviz_mclust.R#L46)
[fviz_dend.R:332](https://github.com/kassambara/factoextra/blob/1689fc74b938510b773e4a595999e82bdf37104e/R/fviz_dend.R#L332)
[fviz_dend.R:339](https://github.com/kassambara/factoextra/blob/1689fc74b938510b773e4a595999e82bdf37104e/R/fviz_dend.R#L339)

**Comment by HedvigS** [comment]
Yes please, this would be great if it could be addressed. Wrapping the relevant part of our scripts in suppressWarnings() in the meantime.

**Comment by ffsammak** [comment]
I keep getting the same message: `guides(<scale> = FALSE)` is deprecated. Please use `guides(<scale> = "none")` instead.

**Comment by erdeyl** [comment]
For users experiencing this issue, a temporary workaround is available via the `factoextraplus` package ([![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.18131494.svg)](https://doi.org/10.5281/zenodo.18131494)), which includes this fix along with other compatibility updates for ggplot2 3.5.2+ and R 4.1.0+.

```r
remotes::install_github("erdeyl/factoextraplus")
```

The fix replaces `guides(... = FALSE)` with `guides(... = "none")` in `fviz_dend()` as required by ggplot2 3.3.4+.

A pull request with these fixes has been submitted (#194). Once merged, users can switch back to the original factoextra package.
